cli/dcoscli/cluster/main.py

```diff
@@ -123,14 +123,19 @@ def _list(json_, attached):
     return
 
 
-def _remove(name):
+def _remove(name, all=False):
     """
     :param name: name of cluster
     :type name: str
+    :param all: remove all clusters if True
+    :type all: bool
     :rtype: None
     """
-
-    return cluster.remove(name)
+    if all:
```

Comment:
`all` is a Python built-in function and so it should not be used as a variable or parameter name. 

❓Also, I find it weird to have a function take `name` and `all` - maybe `Name` can be made to be an `Optional[str]` and if `all` is `True` and `name` is not `None` then an error is raised.